### DISCUSSION TITLE
Grouped accessories with "es" from different devices

### DISCUSSION THREAD

**Discussion by Hanimaniac** [RavenSystem/esp-homekit-devices#1791] [General]
Hello Community,

is it possible to add accessories from another device (e.g. shelly i3) to the "es" array of an device (e.g. wemos d1 mini)?
In my case, I have a irrigation system with 6 water valves divided up in 2 shelly rgbw2 ( one shelly full with 4 valves and the second with 2 valves).
Would it be possible to group all 6 valves in Homekit into one group?
Thanks
Hanimaniac

**Comment by RavenSystem** [comment]
It is not possible.
